spec/index.md

```diff
@@ -2330,6 +2330,34 @@ This function returns fee percentiles, measured in millisatoshi/vbyte (1000 mill
 
 The [standard nearest-rank estimation method](https://en.wikipedia.org/wiki/Percentile#The_nearest-rank_method), inclusive, with the addition of a 0th percentile is used. Concretely, for any i from 1 to 100, the ith percentile is the fee with rank `⌈i * 100⌉`. The 0th percentile is defined as the smallest fee (excluding coinbase transactions).
 
+### IC method `take_snapshot` {#ic-take_snapshot}
+
+This method takes a snapshot of the canister. Subsequent `take_snapshot` calls will replace the existing snapshot with a new one.
+
+The optional `duration` parameter can be used to limit the storage duration of a snapshot. When provided, the snapshot will be stored for the specified duration (in seconds) and will be deleted automatically at the end of this duration. In this case,
```

Comment:
Is this feature necessary for the V1 of snapshotting? I think the concept of a duration for deleting snapshots may also clash with the ongoing work on pre-emptive storage charging that's being done by the runtime team, so it might be better to avoid it for now and potentially introduce it later if the need arises.

Reply:
Yes, that is the case. I discussed with Ulan about this today. So I removed it. I will update the doc about the behavior considering the `reserve_cycles.`